test/Microsoft.Azure.Devices.Edge.Test.TempSensor/Program.cs

```diff
@@ -110,4 +116,26 @@ protected override ValidationResult IsValid(object value, ValidationContext cont
             return ValidationResult.Success;
         }
     }
+
+    [AttributeUsage(AttributeTargets.Class)]
+    public sealed class InstallerPathIsRequiredOnWindowsInvalidOnLinux : ValidationAttribute
```

Comment:
Nit: Simplify the name, for example ValidateInstallerPath or CheckIfInstallerPathIsRequired, etc.

Reply:
I was just following the pattern from the [`CommandLineUtils` samples](https://github.com/natemcmaster/CommandLineUtils/blob/e6e0a36ffeecbfabb84546a3f9bab4f053aa8cb7/docs/samples/validation/attributes/Program.cs#L78). I could shorten the name, but if you see a class with the attribute `[ValidateInstallerPath]` what does that really tell you? You'd have to go read the attribute impl code to find out. I agree that `[InstallerPathIsRequiredOnWindowsInvalidOnLinux]` is ugly, but the real problem is the lack of expressive syntax for this sort of thing in `CommandLineUtils`. E.g. there's no way to say that an `Option` is specific to a certain platform.

Reply:
My concern is what if we support more platforms and we can't put all platform names to construct an attribute name.

Reply:
When those platforms come along we can easily change the name. Would you prefer something like `InstallPathIsRequiredOnWindowsInvalidOtherwise`?

Reply:
Since we can't find a perfect name for it now and it is just internal code; I am ok to leave it now.